exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -179,37 +181,42 @@ public boolean putIndexTemplate(
     template.put("index_patterns", Collections.singletonList(templateName + INDEX_DELIMITER + "*"));
 
     // update alias in template in case it was changed in configuration
-    template.put("aliases", Collections.singletonMap(aliasName, Collections.EMPTY_MAP));
-
-    final PutIndexTemplateRequest request =
-        new PutIndexTemplateRequest(templateName).source(template);
+    template.put("aliases", Collections.singletonMap(aliasName, Collections.emptyMap()));
 
-    return putIndexTemplate(request);
+    return putIndexTemplate(templateName, template);
   }
 
   /** @return true if request was acknowledged */
-  private boolean putIndexTemplate(final PutIndexTemplateRequest putIndexTemplateRequest) {
+  private boolean putIndexTemplate(final String templateName, final Object body) {
     try {
-      return client
-          .indices()
-          .putTemplate(putIndexTemplateRequest, RequestOptions.DEFAULT)
-          .isAcknowledged();
+      final var request = new Request("PUT", "/_template/" + templateName);
+      request.addParameter("include_type_name", "true");
+      request.setJsonEntity(MAPPER.writeValueAsString(body));
+
+      final var response = client.performRequest(request);
+      final var statusLine = response.getStatusLine();
+      final var status = statusLine.getStatusCode();
+      if (status >= 400) {
+        log.warn("Failed to put index template {}", statusLine.getReasonPhrase());
```

Comment:
I'm wondering whether this should be warn log or an ElasticsearchExporterException.

Reply:
Hm, you're probably right; I hate throwing exceptions, but we should be consistent here until we actually use either everywhere ;)

Reply:
Well, there might be a good reason to just log here, although this feels like an error state. I have not thought it through. We should consider our options here before choosing consistency. When does this occur and what happens to the system when we throw an exception here. What was the original behaviour?

Reply:
That's very good point - after a bit of digging, I realize I missed it originally as well. The response from the server contains a member, acknowledged, which is expected to be returned here. I also verified HTTP errors (e.g. 4xx and 5xx) actually cause an IOException to be thrown when performing the request, so there isn't anything to check at all. Good catch :muscle:

Reply:
Nice, great work! Thanks for diving into this.